File: apps/institution-api/Models/VwSpecializationPrograms.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwSpecializationPrograms
    {
        public string CipSubSeriesCode { get; set; }
        public List<VwProgram> Programs { get; set; }
    }
}

File: libs/common/models/vw-specialization-programs.ts


import { VwProgram } from './vw-program';

export class VwSpecializationPrograms {
    public cipSubSeriesCode: string = '';
    public programs: VwProgram[] = [];

    constructor(init? : Partial<VwSpecializationPrograms>) {
        
        Object.assign(this, init);
    }
}

